test/E2E/TestFunctionApp/extensions.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Do not check in this file. Use `func extensions install` command. This command will generate extensions.csproj with the latest nuget packages.

Reply:
How does it get generated? Does it look at the bindings?

Reply:
func cli parses function.json and generates extensions.csproj